sklearn/cluster/tests/test_k_means.py

```diff
@@ -771,4 +774,85 @@ def test_x_squared_norms_init_centroids():
 def test_max_iter_error():
 
     km = KMeans(max_iter=-1)
-    assert_raise_message(ValueError, 'Number of iterations should be', km.fit, X)
+    assert_raise_message(ValueError, 'Number of iterations should be',
+                         km.fit, X)
+
+
+def test_kmeans_float32_64():
```

Comment:
I now named the new test function as `test_float_precision()` since it tests both `KMeans` and `MiniBatchKMeans`.

@MechCoder I guess `int` input data should be tested in other functions since its clustering results will not remain the same as `float` data here?